test/iostream.jl

```diff
@@ -0,0 +1,19 @@
+@test mktemp() do _, file
+    skipchars(file, isspace)
+    true
+end
+
+@test eof(skipchars(IOBuffer("    "), isspace))
+@test eof(skipchars(IOBuffer("#    \n   "), isspace, linecomment='#'))
+
+macro test_skipchars(str, expected_char, lnc=Char(0xffffffff))
+  quote
```

Comment:
Ah gawd, I knew I should have used a automated checker or something. I remember that last time I made a PR incorrect whitespace made the tests fail (and I tested [here](https://github.com/fcard/julia/pull/5) (with a failure on OSX 'coz I am poor)), was that disabled at some point?

Either way I will fix ASAP. Does [ci skip] still work?